src/plugins/plot/inspector/forms/YAxisForm.vue

```diff
@@ -180,7 +180,10 @@ export default {
             this.autoscale = this.yAxis.get('autoscale');
             this.autoscalePadding = this.yAxis.get('autoscalePadding');
             const range = this.yAxis.get('range');
-            if (range) {
+            if (!range) {
+                this.rangeMin = 0;
```

Comment:
Why do we need to default these values when we didn't before? Also, why is `0` the right value to use here? Not saying it's wrong, just want to understand it before we merge.

Reply:
If range comes back undefined, we need to set the rangeMax and rangeMin to something. Otherwise it would remember the previous rangeMax and rangeMin. That's the only reason. I didn't see a bug around it, It was just something I noticed and guarded for.

Reply:
OK I'll change it to undefined.